src/app/public/modules/ui-config/ui-config.service.ts

```diff
@@ -0,0 +1,21 @@
+import {
+  Injectable
+} from '@angular/core';
+
+import {
+  Observable
+} from 'rxjs';
```

Comment:
Is it “safe” to import from rxjs blindly? I seem to remember an article about doing this in  rxjs version 4 but could be wrong…
```
import {
  Observable
} from 'rxjs'; // Should be `rxjs/Observable`?
```

Reply:
Related: https://github.com/angular/angular/issues/20349

Reply:
Fixed.